src/com/ibm/janusgraph/utils/generator/CSVGenerator.java

```diff
@@ -193,8 +173,7 @@ public void writeEdgeCSVs(EdgeTypeBean type, String outputDirectory ){
                 System.out.println("Generated edge file: "+ csvFile);
             }
         } catch (Exception e) {
-            // TODO Auto-generated catch block
-            throw new RuntimeException(e.toString());
+            e.printStackTrace();
```

Comment:
Do you want to hide the exception? Maybe consider to throw after you print the stack track?